lib/command/src/Obelisk/Command.hs

```diff
@@ -220,20 +221,35 @@ thunkDirectoryParser = fmap (dropTrailingPathSeparator . normalise) . strArgumen
   , help "Path to directory containing thunk data"
   ]
 
+thunkConfig :: Parser ThunkConfig
+thunkConfig = ThunkConfig
+  <$>
+    (   flag' (Just True) (long "private" <> help "Mark thunks as pointing to a private repository")
+    <|> flag' (Just False) (long "public" <> help "Mark thunks as pointing to a public repository")
+    <|> pure Nothing
+    )
```

Comment:
Shouldn't we fail parsing if both private and public are provided? This seems like it will pick the first, which could allow someone to accidentally add `public` rather than replace `private` after open-sourcing a repo.
Replacing the order here could then potentially break things

Reply:
this works as you intended, if private and public are both provided, it marks the last one as invalid, because it is not consumed. although, the error message is just that the last one is an invalid option, which might be worth changing